src/test/scala/org/broadinstitute/dsde/rawls/graph/OrientDbTestFixture.scala

```diff
@@ -14,6 +18,49 @@ trait OrientDbTestFixture extends BeforeAndAfterAll {
   lazy val dataSource = DataSource("memory:"+testDbName, "admin", "admin")
   lazy val graph = new OrientGraph("memory:"+testDbName)
   lazy val txn: RawlsTransaction = new RawlsTransaction(graph, dataSource)
+  lazy val workspaceDAO: GraphWorkspaceDAO = new GraphWorkspaceDAO()
+  lazy val entityDAO: GraphEntityDAO = new GraphEntityDAO()
+
+  def initializeTestGraph(tx:RawlsTransaction = txn): Unit = {
```

Comment:
This isn't called by default, but calling it within a test class will dump a reasonable-looking workspace into your test environment for you to fiddle with.